proxy/src/session.rs

```diff
@@ -266,7 +276,7 @@ pub mod settings {
     /// `CoCo` config parameters subject to user preferences
     #[derive(Debug, Deserialize, Serialize, PartialEq)]
     pub struct CoCo {
-        /// Sources of feeds
+        /// Peers to connect to at startup.
```

Comment:
Is this any kind of peer or specifically seed nods?

Reply:
Any peer.

Reply:
Then let's rename the field accordingly to reflect that.

Reply:
It already has the correct name, it was the "feed" part that was incorrect.

Reply:
The field is still called `seeds` and not `peers`.